src/compute/src/rpc/service/exchange_metrics.rs

```diff
@@ -31,6 +32,14 @@ impl ExchangeServiceMetrics {
         )
         .unwrap();
 
+        let stream_fragment_exchange_bytes = register_int_counter_vec_with_registry!(
```

Comment:
Why send and recv metrics are defined in two places? This would make things a little bit confusing...

Reply:
I follow the PR about the actor exchange, the exchange and receive are defined in two places.